config.json

```diff
@@ -2,7 +2,13 @@
     "docs": {
         "gdoc1": {
             "fun": "sendMessage",
-            "dest": "1w4LBF6wbRNVynAk8cQURyz8yZCbTw5hPwcqr87S46hY"
+            "dest": "1w4LBF6wbRNVynAk8cQURyz8yZCbTw5hPwcqr87S46hY",
+            "room": "Datopian"
+        },
+        "gdoc2": {
```

Comment:
Is there really good reason naming these `gdoc1` and `gdoc2`. Think would be much useful naming them with reasonable names Eg: `gdocDatopian` and `gdocArtErthTech` etc...
Same with Gist

Reply:
ok, will be fixed

Reply:
@zelima but in order to distinguish, I added room name. But anyway, I will change it